plio/tests.py

```diff
@@ -288,6 +288,127 @@ def test_duplicate(self):
         self.assertNotEqual(plio.uuid, response.data["uuid"])
         self.assertEqual(plio.name, response.data["name"])
 
+    def test_default_ordering_when_no_ordering_specified(self):
+        # create a third plio
+        Plio.objects.create(name="Plio 3", video=self.video, created_by=self.user)
+
+        # make a request to list the plio uuids without specifying any order
+        # NOTE: default ordering should come out to be '-updated_at'
+        # order of plios should be [plio_3, plio_2, plio_1]
+        response = self.client.get("/api/v1/plios/list_uuid/")
+        self.assertListEqual(
+            [plio.uuid for plio in Plio.objects.order_by("-updated_at")],
```

Comment:
I think we should specifically check 3,2,1 here and 1,2,3 there - or else, there could be an error in both Plio.objects.order_by and response.data - and so, both of them are equal but not equal to the value that should be.